packages/quantic/force-app/main/default/lwc/exampleSearch/exampleSearch.html

```diff
@@ -1,13 +1,13 @@
 <template>
     <div onregisterresulttemplates={handleResultTemplateRegistration}>
         <c-quantic-search-interface engine-id={engineId}>
-            <div class="slds-grid slds-gutters slds-wrap main">
+            <div class="slds-grid slds-gutters slds-wrap main slds-grid_align-center">
                 <div class="slds-col slds-order_2 slds-large-order_1 slds-size_1-of-1 slds-large-size_4-of-12">
                     <c-quantic-facet field="objecttype" label="Type" engine-id={engineId}></c-quantic-facet>
                     <c-quantic-facet field="filetype" label="File Type" engine-id={engineId}></c-quantic-facet>
                     <c-quantic-numeric-facet field="ytlikecount" label="Youtube Likes" engine-id={engineId}></c-quantic-numeric-facet>
                     <c-quantic-date-facet field="date" label="Date" engine-id={engineId}></c-quantic-date-facet>
-                    <c-quantic-category-facet field="geographicalhierarchy" label="Country" engine-id={engineId}></c-quantic-category-facet>
+                    <c-quantic-category-facet field="atlgeographicalhierarchy" label="Country" engine-id={engineId}></c-quantic-category-facet>
```

Comment:
Since you reverted the facet delimiter change, I guess the category facet is not working with the `atlgeographicalhierarchy` field. Shouldn't you revert to the `geographicalhierarchy` field as well?